utils/metadata_generator.py

```diff
@@ -17,19 +17,32 @@ def extract_macros_params(line, macros_name):
         return extracted
     return None
 
-def parse_declare_saved_begin(file, class_and_base, filename):
+
+def parse_file(full_path: str) -> Union[dict, None]:
+    with open(full_path, 'r') as file:
+        for line in file:
+            result = extract_macros_params(line, "DECLARE_SAVEABLE")
+            if result:
+                data = parse_declare_saved_begin(file, result, full_path)
+                return data
+        return None
```

Comment:
Seems like I fixed it, metadata.json lines count changed from 500~ to 900~ and I can definitely see all classes from example file in json output.